bc_obps/registration/migrations/0001_initial.py

```diff
@@ -0,0 +1,308 @@
+                    models.ForeignKey(
+                        db_comment='',
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='user_operators_proof_of_authority',
+                        to='registration.document',
+                    ),
+                ),
+                (
+                    'signed_statuatory_declaration',
+                    models.ForeignKey(
+                        db_comment='',
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='user_operators_signed_statuatory_declaration',
+                        to='registration.document',
+                    ),
+                ),
+                (
+                    'users',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name='user_operators',
+                        to='registration.user',
+                    ),
+                ),
+            ],
+            options={
+                'db_table_comment': 'Through table to connect Users and Operators and track access requests',
+            },
+        ),
+        migrations.AddField(
+            model_name='operator',
+            name='operators',
+            field=models.ManyToManyField(
+                related_name='operator_users', through='registration.UserOperator', to='registration.user'
+            ),
+        ),
+        migrations.AddField(
+            model_name='operator',
+            name='parent_operator',
+            field=models.ForeignKey(
+                blank=True,
+                db_comment='',
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='parent',
+                to='registration.operator',
+            ),
+        ),
+        migrations.CreateModel(
+            name='Operation',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(db_comment='', max_length=1000)),
+                ('operation_type', models.CharField(db_comment='', max_length=1000)),
+                ('eligible_commercial_product_name', models.CharField(db_comment='', max_length=1000)),
+                ('permit_id', models.CharField(db_comment='', max_length=1000)),
+                ('npr_id', models.CharField(blank=True, db_comment='', max_length=1000, null=True)),
+                ('ghfrp_id', models.CharField(blank=True, db_comment='', max_length=1000, null=True)),
+                ('bcghrp_id', models.CharField(blank=True, db_comment='', max_length=1000, null=True)),
+                ('petrinex_id', models.CharField(blank=True, db_comment='', max_length=1000, null=True)),
+                ('location', django.contrib.gis.db.models.fields.PointField(db_comment='', geography=True, srid=4326)),
```

Comment:
Lets either use a non-postgis point type or split the location into decimal lat/long values here

Reply:
It's going to have to be decimal, looks like: https://stackoverflow.com/questions/50155776/pointfield-does-not-become-point-but-geometry